# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pingus/pingus-0.7.2-gcc44.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -up ./src/input/evdev_device.cpp.orig ./src/input/evdev_device.cpp
--- ./src/input/evdev_device.cpp.orig	2007-10-01 19:24:54.000000000 +0200
+++ ./src/input/evdev_device.cpp	2009-02-19 16:35:26.000000000 +0100
@@ -25,6 +25,8 @@
 #include <stdexcept>
 #include <iostream>
 #include <errno.h>
+#include <cstring>
+#include <cstdio>
 
 #include "evdev_device.hpp"
 
diff -up ./src/tinygettext/po_file_reader.cpp.orig ./src/tinygettext/po_file_reader.cpp
--- ./src/tinygettext/po_file_reader.cpp.orig	2007-10-18 19:54:01.000000000 +0200
+++ ./src/tinygettext/po_file_reader.cpp	2009-02-19 15:11:32.000000000 +0100
@@ -26,6 +26,7 @@
 #include <string.h>
 #include "iconv.hpp"
 #include "po_file_reader.hpp"
+#include <cstdio>
 
 namespace TinyGetText {
 
